hscontrol/policy/v2/policy.go

```diff
@@ -66,9 +68,12 @@ func (pm *PolicyManager) updateLocked() (bool, error) {
 	}
 
 	filterHash := deephash.Hash(&filter)
-	filterChanged := filterHash == pm.filterHash
+	filterChanged := filterHash != pm.filterHash
```

Comment:
I see that only SSH tests in v2 fail now, I suspect that it is because there was a bug here that never returned "early" on line 102, causing 110 (clear ssh) to always run. Now that it does return early, I suspect that my logic for when the ssh needs to be clear is wrong, and it wasnt discovered because of this.

Reply:
That a good point, I moved the clear up to a defer so it always runs and rebased. If that makes the tests happy, then I will merge it!